core/pypika_utils.py

```diff
@@ -167,12 +167,140 @@ def filter(self, filters):
         filter_operations = [filter_mapping[filter["func"]](getattr(
             self.current_dataset, filter["field"]), filter["value"]) for filter in filters]
         filter_operations_or = reduce(operator.or_, filter_operations)
+
+        print('filter query string is')
+        print(filter_operations_or)
```

Comment:
I really should stop seeing these in your code ...

Reply:
Removed debugging code